.github/workflows/prettify.yml

```diff
@@ -3,6 +3,16 @@ name: Prettifying
 on:
   pull_request:
     branches: [main]
+    paths-ignore:
+      - "publication/**"
+      - "errata.html"
+      - "errata11.html"
+      - "ontology/*.html"
+      - "index.html"
+      - "validation/tm-json-schema-validation.json"
```

Comment:
Not sure about this either and if we exclude this file we should exclude https://github.com/w3c/wot-thing-description/blob/main/validation/td-json-schema-validation.json also

Reply:
So the initial decision was to format only what we write and not format auto generated stuff. That is why this and index.html are ignored

Reply:
See also https://github.com/w3c/wot-thing-description/issues/1926#issue-2006498787

These two comments should address all the questions

Reply:
I see the rational. Anyhow, the consequence is that once we update the tool that generates those other files might create huge diffs even though it didn't really change that much. Anyhow, I don't feel very strongly about it for now.

Reply:
> we would directly not commit any generated file to the tree

We have to submit at least the index.html but the rest can live somewhere else.

> Anyhow, the consequence is that once we update the tool that generates those other files might create huge diffs even though it didn't really change that much

That is a good point. I am fine to format those as well, it wouldn't hurt to do so.